docs/modules/mapstore/pages/configuring-a-generic-mapstore.adoc

```diff
@@ -257,7 +257,7 @@ mapConfig.setMapStoreConfig(mapStoreConfig);
 ====
 
 |[[id-column]]`id-column`
-|Name of the column that contains the primary key.
+|Name of the column that contains the primary key. A column without a primary key constraint should not be specified as the id-column.
```

Comment:
```suggestion
|Name of the column that contains the primary key. Do not specify a column without a primary key constraint as the id-column.
```

Reply:
Can we make the statement stronger, as suggested above, or is it only a recommendation (as implied by "should")?

Reply:
Yes it would be better like this.